src/test/java/com/meilisearch/sdk/http/URLBuilderTest.java

```diff
@@ -64,6 +66,44 @@ void addParameterStringArray() {
                 classToTest.getParams().toString());
     }
 
+    @Test
+    void addParameterStringIntArray() {
+        classToTest.addParameter("parameter1", new int[] {1, 2});
+        assertEquals("?parameter1=1,2", classToTest.getParams().toString());
+
+        classToTest.addParameter("parameter2", new int[] {3, 4});
+        assertEquals("?parameter1=1,2&parameter2=3,4", classToTest.getParams().toString());
+
+        classToTest.addParameter("parameter3", new int[] {5, 6});
+        assertEquals(
+                "?parameter1=1,2&parameter2=3,4&parameter3=5,6",
+                classToTest.getParams().toString());
+    }
+
+    @Test
+    void addParameterStringDate() {
+        Date date = new Date();
```

Comment:
In this case you can setup any fixed date, and then assert it by a string literal. Your test will be more straightforward.